### ISSUE TITLE
Broken syntax highlighting

### ISSUE THREAD

**Issue by hupfdule** [JalaiAmitahl/maven-compiler.vim#5]
The syntax highlighting is broken.

See below screenshot. There are several problems:

- on line 25: The opening and closing tags for `issueManagement` aren't highlighted correctly
- on line 39: The opening and closing tags inside the `properties` tags aren't highlighted correctly
- on line 53: The whole `dependency` block is commented out, but the contained tags are highlighted as not-commented. Also the commented out starting and closing tags are not highlighted as commented (which should look like line 69)
- on line 69: The lines after this comment with the same indent are incorrectly highlighted

![apron](https://user-images.githubusercontent.com/36069345/55788528-c78e8800-5ab8-11e9-88d6-c640b3a62272.png)

**Comment by amplexdenmark** [comment]
Yeah. I was bitten by the block-comment issue too, but i seriously need the compiler integration.
As a workaround, I commented out the two lines in ftdetect/pom.vim & relies on the plugin manager (ie. git) to preserve this local change
